OrleansDashboard/Implementation/DashboardTelemetryExporter.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Collections.Concurrent;
+using System.Globalization;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using Orleans;
+using Orleans.Concurrency;
+using Orleans.Runtime;
+using OrleansDashboard.Model;
+
+namespace OrleansDashboard.Implementation
+{
+    public class DashboardTelemetryExporter : BaseExporter<Metric>
+    {
+        public readonly struct Value<T>
+        {
+            public readonly T Current;
+            public readonly T Last;
+
+            public Value(T value)
+                : this(value, value)
+            {
+            }
+
+            public Value(T last, T current)
+            {
+                Last = last;
+
+                Current = current;
+            }
+
+            public Value<T> Update(T newValue)
+            {
+                return new Value<T>(Current, newValue);
+            }
+        }
+
+        private readonly ConcurrentDictionary<string, Value<double>> metrics = new ConcurrentDictionary<string, Value<double>>();
```

Comment:
Is this needed? It should be a normal dictionary in a local variable now.

Reply:
True, I was contemplating the same but ended up just reusing the code from the old consumer. Corrected it now :)